code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -409,6 +409,7 @@ MATCHBOXES ARE ALSO IN FANCY.DM
 	item_state = "cigar"
 	smoketime = 1500
 	chem_volume = 20
+	species_fit = list(VOX_SHAPED)
```

Comment:
species_fit seems to determine if the clothing has a different sprite for other races.
You must be thinking of species_allowed